### ISSUE TITLE
How to make slight modifications to the lstn/dataloaders/train_datasets.py?

### ISSUE THREAD

**Issue by qian507** [harrytea/Detect-AnyShadow#2]
Hello author, how do I need to modify this aspect?
![图片](https://github.com/harrytea/Detect-AnyShadow/assets/58128977/c5370d66-632a-4263-9d51-3f53cd54a168)
![图片](https://github.com/harrytea/Detect-AnyShadow/assets/58128977/54df880e-b334-42b4-8af7-7ec40fff1495)

**Comment by harrytea** [comment]
Replace the lines in the py file where images and masks are loaded with the image loading method from OpenCV.

For example, replace line 55 with `ref_label =  cv2.imread(xxx)`

**Comment by qian507** [comment]
Thank you very much for your timely reply, but does ref_index need further processing?
![图片](https://github.com/harrytea/Detect-AnyShadow/assets/58128977/80b51f5a-755e-4834-8811-5f3a9d4534dc)
